### ISSUE TITLE
How to use iconify with tabler?

### ISSUE THREAD

**Issue by argo-uka** [unocss/unocss#1903]
I read hundred times the readme file, but I don't understand how to make it work with tabler-icons.
I'm using unocss with solidjs/vite. 

First I installed the dependecies
```sh
npm i -D @unocss/preset-icons @iconify-json/tabler
```

This is my unocss config
```js
import { defineConfig } from "@unocss/vite";
import presetUno from "@unocss/preset-uno";
import presetIcons from "@unocss/preset-icons";

import transformerDirective from "@unocss/transformer-directives";

export default defineConfig({
  presets: [
    presetIcons({
      /* options */
    }),
    presetUno(),
  ],
  transformers: [transformerDirective()],
});
```
Then I inserted the icons in one of my component
```jsx
import { onMount } from "solid-js";

export function Panel({ children, header, observerCallback }) {
  let container;
  let observer;
  let headerElement;
  let bodyElement;

  onMount(() => {
    const observerOptions = {
      childList: false,
      attributes: true,
      subtree: false,
    };
    observer = new MutationObserver((mutationList) => {
      mutationList.forEach((mutation) => {
        console.log(mutation.type);
        if (mutation.type === "attributes") {
          if (typeof observerCallback === "function") {
            observerCallback(
              container.clientHeight - headerElement.offsetHeight - 1
            );
          }
        }
      });
    });
    observer.observe(container, observerOptions);
  });

  return (
    <div class="flex-auto p-0 m-0 overflow-y-hidden" ref={container}>
      <div
        ref={headerElement}
        class="font-bold uppercase px-4px py-4px font-small bg-stone-50 border-b border-stone-200"
      >
        <div class="i-tabler:filter"></div>
        <div class="i-tabler-filter"></div>
        {header}
      </div>
      {children}
    </div>
  );
}
```

The panel gets rendered but without the icons?
What am I missing? Can someone, kindly, show me an ELI5?

**Comment by userquin** [comment]
@argo-uka have you checked the solid-js example? Also read solid-js here: https://github.com/unocss/unocss/tree/main/packages/vite#solid

You need to import `uno.css` using virtual module.

**Comment by argo-uka** [comment]
> @argo-uka have you checked the solid-js example? Also read solid-js here: https://github.com/unocss/unocss/tree/main/packages/vite#solid
> 
> You need to import `uno.css` using virtual module in your entry point: https://github.com/unocss/unocss/blob/main/examples/vite-solid/src/index.tsx#L4

Oh yeah, unocss classes  like `px-0 bg-red-100` works. But not the icons. In my `index.jsx` I have
```jsx
...
import "@unocss/reset/tailwind.css";
import "uno.css";
...
```

So, as your suggestion, I imported `uno.css` in the panel component, still doesn't work.
```jsx
import { onMount } from "solid-js";
import "uno.css";

export function Panel({ children, header, observerCallback }) {
  let container;
  let observer;
  let headerElement;
  let bodyElement;

  onMount(() => {
    const observerOptions = {
      childList: false,
      attributes: true,
      subtree: false,
    };
    observer = new MutationObserver((mutationList) => {
      mutationList.forEach((mutation) => {
        console.log(mutation.type);
        if (mutation.type === "attributes") {
          if (typeof observerCallback === "function") {
            observerCallback(
              container.clientHeight - headerElement.offsetHeight - 1
            );
          }
        }
      });
    });
    observer.observe(container, observerOptions);
  });

  return (
    <div class="flex-auto p-0 m-0 overflow-y-hidden" ref={container}>
      <div
        ref={headerElement}
        class="font-bold uppercase px-4px py-4px font-small bg-stone-50 border-b border-stone-200"
      >
        <div class="i-tabler:filter"></div>
        <div class="i-tabler-filter"></div>
        {header}
      </div>
      {children}
    </div>
  );
}

```

**Comment by jd-solanki** [comment]
It will be better if you can paste codesandbox or stackblitz link. Moreover, do styles get generated for your icon classes?

**Comment by antfu** [comment]
We temporarily close this due to the lack of enough information.
Please provide a [minimal reproduction](https://stackoverflow.com/help/minimal-reproducible-example) to reopen the issue.
Thanks.

> [Why reproduction is required](https://antfu.me/posts/why-reproductions-are-required)
